front/src/applications/operationalStudies/components/MacroEditor/ngeToOsrd.ts

```diff
@@ -512,9 +519,11 @@ const handleUpdateTimetableItem = async ({
     schedule,
     // Reset margins because they contain references to path items
     margins: undefined,
-    paced: undefined,
-    exceptions: undefined,
```

Comment:
We still need to reset these two fields. Without this, turning a paced train into a train schedule fails because the backend rejects these two fields.